build/DependencyVersions.props

```diff
@@ -12,12 +12,12 @@
     <!-- We'll usually want to keep these versions in sync, but we may want to diverge in some
          cases, so use separate properties but derive one from the other unless we want to
          explicitly use different versions. -->
-    <CLI_NETSDK_Version>2.0.1-servicing-20170912-1</CLI_NETSDK_Version>
+    <CLI_NETSDK_Version>2.0.2-vspre-20170921-5</CLI_NETSDK_Version>
```

Comment:
We need to update the SDK here to one after 2.0.2-vspre was merged into 2.0.1-servicing.